### DISCUSSION TITLE
Domain names magic

### DISCUSSION THREAD

**Discussion by microspace** [nginx-proxy/nginx-proxy#1952] [Q&A]
I have two site behind nginx-proxy. One is flask-app, the other is wordpress site.
When I enter www.wp-site-domain.ru, browser redirects me to wp-site-domain.ru with 301 redirect
With flask site, when I enter www.flask-site.ru, browser doesn't redirect me. 
[Screenshot](https://cloud.mail.ru/public/5dML/LQUSU1DnN)
**But I used similar docker command to launch both containers!!**
I can't understand where the redirect magic happens!
I launch flask app using [this](https://github.com/jwilder/nginx-proxy/issues/384#issuecomment-193870570) 
`docker run -d --restart=always -e VIRTUAL_HOST=*.flask-site.ru,flask-site.ru myflask`
I launch wordress using this:
`docker run -d --restart=always --name wordpress1 --volumes-from wordpress1_data --link wordpress1_mariadb:mysql -e VIRTUAL_HOST=*.wp-site-domain.ru,wp-site-domain.ru wordpress`

**Comment by microspace** [comment]
At last I should make 301 redirection from www to nonwww?
But why wordpress site makes redirection is still mysterious for me.

**Comment by helderco** [comment]
Inspect your `/etc/nginx/conf.d/default.conf` file produced by docker-gen in nginx-proxy.

**Comment by microspace** [comment]
https://gist.github.com/microspace/60c7b4e59136a529d316
No clue to who does the redirection
